app/Filament/Resources/TaskResource.php

```diff
@@ -96,30 +96,30 @@ public static function form(Form $form, bool $serverSelect = true): Form
                                 Frequency::MONTHLY => 'Monthly',
                                 Frequency::YEARLY => 'Yearly',
                             ])
-                            ->formatStateUsing(fn(?Task $record): int => $record?->frequency ?? Frequency::DAILY)
+                            ->formatStateUsing(fn (?Task $record): int => $record?->frequency ?? Frequency::DAILY)
                             ->required()
                             ->live()
                             ->native(false),
                         TextInput::make('interval')
                             ->prefix('Every')
-                            ->suffix(fn(Get $get): string => match ((int)$get('frequency')) {
-                                    Frequency::SECONDLY => 'second',
-                                    Frequency::MINUTELY => 'minute',
-                                    Frequency::HOURLY => 'hour',
-                                    Frequency::DAILY => 'day',
-                                    Frequency::WEEKLY => 'week',
-                                    Frequency::MONTHLY => 'month',
-                                    Frequency::YEARLY => 'year',
-                                } . '(s)')
+                            ->suffix(fn (Get $get): string => match ((int) $get('frequency')) {
+                                Frequency::SECONDLY => 'second',
+                                Frequency::MINUTELY => 'minute',
+                                Frequency::HOURLY => 'hour',
+                                Frequency::DAILY => 'day',
+                                Frequency::WEEKLY => 'week',
+                                Frequency::MONTHLY => 'month',
+                                Frequency::YEARLY => 'year',
+                            }.'(s)')
```

Comment:
how do i fix this as a standard? I want spaces around concatenation (maybe all) operators
```suggestion
                            } . '(s)')
```